integration-test/src/test/java/org/apache/iotdb/util/AbstractSchemaIT.java

```diff
@@ -57,7 +57,7 @@ public abstract class AbstractSchemaIT {
 
   @Parameterized.Parameters(name = "SchemaEngineMode={0}")
   public static Iterable<SchemaTestMode> data() {
-    return Arrays.asList(SchemaTestMode.Memory, SchemaTestMode.PBTree);
+    return Arrays.asList(SchemaTestMode.Memory);
```

Comment:
Why removing `PBTree` mode.

Reply:
For local testing convenience. Already added back.